network/p2p/client.go

```diff
@@ -40,11 +41,15 @@ type CrossChainAppResponseCallback func(
 )
 
 type Client struct {
-	handlerID     uint64
-	handlerPrefix []byte
-	router        *router
-	sender        common.AppSender
-	options       *clientOptions
+	handlerID                      uint64
```

Comment:
q: why is Client constructor not in this file? It seems to be in network.go file.